src/core/actions/search.js

```diff
@@ -5,10 +5,10 @@ import {
 } from 'core/constants';
 
 
-export function searchStart({ filters, page, results }) {
+export function searchStart({ errorHandlerId, filters, page, results }) {
```

Comment:
Since all of these parameters are required, the function should throw an error if any are empty. This is important because action creators are one part of the redux flow that we can't capture and replay. In other words, if there is a bug in some calling code where it's passing an empty parameter to an action creator, it might be hard to track down.